docs/main/Solutioning/backup-and-restore/kasten-br-fs.md

```diff
@@ -0,0 +1,338 @@
+test-cd9847c9c-wc6th   1/1     Running   0          25s
+```
+
+**Command**
+
+```
+kubectl get pvc -n test
+```
+
+**Output**
+
+```
+NAME           STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS                  AGE
+mayastor-pvc   Bound    pvc-b5baa4cf-b126-42e6-b11f-3e20aeb3ab7b   1Gi        RWO            mayastor-thin-multi-replica   54s
+```
+
+**Sample Data**
+
+```
+kubectl exec -it test-cd9847c9c-wc6th -n test -- bash
+root@test-cd9847c9c-wc6th:/# cd volume/
+root@test-cd9847c9c-wc6th:/volume# cat abc
+Mayastor Kasten backup and restore
+root@test-cd9847c9c-wc6th:/volume#
+```
+
+#### Applications from Kasten Dashboard
+
+By default, the Kasten platform equates namespaces to applications. Since we have already installed the applications, clicking the Applications card on the dashboard will take us to the following view:
+
+![test-applications](../../assets/test-applications.png)
+
+#### Create Policies from Kasten Dashboard
+
+Policies are implemented in Kasten to streamline/automate data management workflows. A section on the management of policies is located adjacent to the Applications card on the primary dashboard. To accomplish this, they integrate actions that you wish to execute (e.g., snapshots), a frequency or schedule for how often you want to take that action, and selection criteria for the resources you want to manage.
+
+:::important
+Policy can be either created from this page or from the application page. Users should “Enable backups via Snapshot exports” to export applications to s3.
+:::
+
+- Select the application. In this example, it is “test”.
+
+![new-policy-test](../../assets/new-policy-test.png)
+
+- Click **Create Policy** once all necessary configurations are done.
+
+![create-policy-test](../../assets/create-policy-test.png)
+
+- Click **Show import details** to get the import key. Without the import key, users are unable to import this on the target cluster. 
+
+![show-import-details](../../assets/show-import-details.png)
+
+- When you create an import policy on the receiving cluster, you need to copy and paste the encoded text that is displayed in the **Importing Data** dialog box.
+
+![importing-data](../../assets/importing-data.png)
+
+#### Backup
+
+Once the policies have been created, it is possible to run the backup. In this scenario, we have created policies to run “on-demand”. A snapshot can be scheduled based on the available options. Example: hourly/weekly
+
+- Select **Run Once > Yes, Continue** to trigger the snapshot. Backup operations convert application and volume snapshots into backups by transforming them into an infrastructure-independent format and then storing them in a target location (Google Cloud Bucket).
+
+![run-once](../../assets/run-once.png)
+
+The snapshots status can be monitored and exported from the Dashboard. The backup had been successfully completed and exported to the storage location. 
```

Comment:
"status can be monitored and exported" how can status be exported?

Reply:
@balaharish7 rephrase this as in confluence. 

You can monitor the status of the snapshots and export from Dashboard. we could see the backup completed and exported to our storage location.

@avishnu We have an image attached which shows how this can be exported as part of backup steps.